tests/helpers/graphical_units.py

```diff
@@ -5,6 +5,8 @@
 
 import os
 import tempfile
+from operator import truediv
```

Comment:
I think this does not need to be added here?
```suggestion
```

Reply:
Fixed! I think that was a result of one of my VSCode extensions adding that, next time I'll be more careful when adding changes